src/blocks/types.ts

```diff
@@ -18,6 +18,8 @@ export const EditableKINDS = new Set([
   KINDS.Notebook,
   KINDS.Content,
   KINDS.Navigation,
+  KINDS.Image,
+  KINDS.Output,
```

Comment:
Where is editable kinds used? I think I thought these were ones with drafts, which these still don't have.

Reply:
it's used here: https://github.com/curvenote/platform/blob/dev/packages/app/src/components/Page/Block/NavActions.tsx#L70

To enable the edit button but if the enum should mean editable-because-has-draft I can revert the change here and replace that line in NavAction with a `switch(kind)`?

That's maybe better to keep this type as intended.

Reply:
And strictly speaking we don;t need image quite yet, I was just keeping top-level images along for the ride as far as I could